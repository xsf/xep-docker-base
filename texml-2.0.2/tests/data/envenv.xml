<TeXML><env name="n1" begin="b1" end="e1">
<cmd name="par1"/>
  <env name="n2" begin="b2" end="e2">
    <env name="n3" begin="b3" end="e3"/>
</env></env>
</TeXML>
