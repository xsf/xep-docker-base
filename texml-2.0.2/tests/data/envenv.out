\b1{n1}
\par1{}
\b2{n2}
\b3{n3}
\e3{n3}
\e2{n2}
\e1{n1}
